tests/system/common/assets/notification_nuclio_function.py

```diff
@@ -31,6 +31,9 @@ def handler(context, event):
     if operation == "add":
         context.user_data.data_list.append(data)
 
+    if runs := event_data.get("runs"):
+        context.user_data.data_list.append(runs)
```

Comment:
is this used at all?..

Reply:
Yes of course, That's how I check at the end that all the runs arrived

Reply:
It just seems that `get_notifications_from_nuclio_and_reset_notification_cache` only does the `list` operation. are you using a different function to check this?

Reply:
Added some clarifications to eliminate confusions there :)